docs/upgrade.html

```diff
@@ -66,12 +66,23 @@
         individual messages is only reduced by the overhead of the batch format. This similarly affects the
         producer's <code>batch.size</code> configuration.</li>
     <li>GC log rotation is enabled by default, see KAFKA-3754 for details.</li>
+    <li>Message protocol introduces support for headers as an array of Header with uftf8 <code>key</code> and byte[] <code>value</code>. 
```

Comment:
Can we be more precise on the message format version? e.g. 0.11.0. Same for the descriptions below, `Older clients` -> `clients before 0.11.0`